.neutrinorc.js

```diff
@@ -17,6 +17,29 @@ module.exports = {
           historyApiFallback: {
             disableDotRule: true,
           },
+          headers: {
+            /*
+              Whenever this changes we will also need to update the headers
+              for our deployed environments in the cloudops repo (TODO: add link).
+
+              script-src: unsafe-eval is only for local dev, because of webpack/hotreload
+              img-src: gravatar & githubusercontent & i1.wp.com for user avatars
+                       (which can't be locked down to a specific path due to CSP format)
+              style-src: https://fonts.googleapis.com for dynamically loaded fonts
+              font-src: https://fonts.gstatic.com for fonts!
+              connect-src: https://localhost:9000 for the backend api
+                           {ws,https}://localhost:8010 for webpack connections (local dev only)
+                           https://balrog-localdev.auth0.com for authentication.
+                           note: this is different in stage/prod
+              frame-src: https://balrog-localdev.auth0.com for background token refreshes
+            */
+            'Content-Security-Policy': "default-src 'none'; script-src 'self' 'unsafe-eval'; img-src 'self' https://*.gravatar.com https://*.githubusercontent.com https://i1.wp.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src https://fonts.gstatic.com; connect-src http://localhost:9000 ws://localhost:9000 https://localhost:8010 https://balrog-localdev.auth0.com; frame-src: https://balrog-localdev.auth0.com",
```

Comment:
For places where we have `localhost:9000`, I think we could just use `self` instead.

Reply:
Good point. I'll do some double checking to make sure this works with ws://, too.